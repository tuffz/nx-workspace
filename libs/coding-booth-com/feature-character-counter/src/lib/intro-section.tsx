export function IntroSection() {
  return (
    <div className="flex flex-col items-center justify-center gap-10 text-center lg:items-start lg:justify-normal lg:gap-14 lg:text-left">
      <h1 className="text-4xl font-extrabold tracking-tight md:-mb-4 lg:text-6xl dark:text-white">
        No more guesswork, count characters in a blink!
      </h1>
      <p className="text-lg leading-relaxed opacity-80 dark:text-white">
        Introducing{' '}
        <strong className="text-xl font-bold tracking-tight md:-mb-4 md:text-2xl lg:text-3xl">
          CharacterCounter
        </strong>{' '}
        - your{' '}
        <strong className="underline decoration-lime-500 decoration-4">
          simple
        </strong>{' '}
        tool for counting characters. It's{' '}
        <strong className="underline decoration-lime-500 decoration-4">
          straightforward
        </strong>
        ,{' '}
        <strong className="underline decoration-lime-500 decoration-4">
          efficient
        </strong>
        , and gets the job done{' '}
        <strong className="underline decoration-lime-500 decoration-4">
          hassle-free
        </strong>
        !
      </p>
    </div>
  );
}
