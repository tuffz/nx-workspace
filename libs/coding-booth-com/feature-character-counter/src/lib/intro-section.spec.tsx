import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import { IntroSection } from './intro-section';

describe('IntroSection', () => {
  test('matches the snapshot', () => {
    const { container } = render(<IntroSection />);
    expect(container).toMatchSnapshot();
  });
});
