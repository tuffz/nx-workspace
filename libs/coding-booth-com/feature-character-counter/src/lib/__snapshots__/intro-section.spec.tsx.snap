// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroSection matches the snapshot 1`] = `
<div>
  <div
    class="flex flex-col items-center justify-center gap-10 text-center lg:items-start lg:justify-normal lg:gap-14 lg:text-left"
  >
    <h1
      class="text-4xl font-extrabold tracking-tight md:-mb-4 lg:text-6xl dark:text-white"
    >
      No more guesswork, count characters in a blink!
    </h1>
    <p
      class="text-lg leading-relaxed opacity-80 dark:text-white"
    >
      Introducing
       
      <strong
        class="text-xl font-bold tracking-tight md:-mb-4 md:text-2xl lg:text-3xl"
      >
        CharacterCounter
      </strong>
       
      - your
       
      <strong
        class="underline decoration-lime-500 decoration-4"
      >
        simple
      </strong>
       
      tool for counting characters. It's
       
      <strong
        class="underline decoration-lime-500 decoration-4"
      >
        straightforward
      </strong>
      ,
       
      <strong
        class="underline decoration-lime-500 decoration-4"
      >
        efficient
      </strong>
      , and gets the job done
       
      <strong
        class="underline decoration-lime-500 decoration-4"
      >
        hassle-free
      </strong>
      !
    </p>
  </div>
</div>
`;
